### PR TITLE
Support Scala Native 0.5.0-RC1

### PR DESCRIPTION
**Pull request by lolgab**
Scala Native `0.5.0-RC1` was just released: https://github.com/scala-native/scala-native/releases/tag/v0.5.0-RC1

Pull Request: https://github.com/com-lihaoyi/mill/pull/3054

### CODE REVIEW

File: scalanativelib/worker/0.5/src/mill/scalanativelib/worker/scala/scalanative/Utils.scala
Comment (lefou) [inline scalanativelib/worker/0.5/src/mill/scalanativelib/worker/scala/scalanative/Utils.scala:1]: Can you fix the mismatch between filename and object name?

**Review by lefou — COMMENTED** [review]
This looks good to me. One nit below.

File: scalanativelib/worker/0.5/src/mill/scalanativelib/worker/scala/scalanative/Utils.scala
Comment (lolgab) [reply scalanativelib/worker/0.5/src/mill/scalanativelib/worker/scala/scalanative/Utils.scala:1]: This file was not needed for Scala Native `0.5`.
But I fixed it for `0.4`.

Comment (lefou) [reply scalanativelib/worker/0.5/src/mill/scalanativelib/worker/scala/scalanative/Utils.scala:1]: Even better.

**Review by lefou — APPROVED** [review]

**Pull request merged**